tests/TestCase/Database/DriverTest.php

```diff
@@ -267,6 +268,17 @@ public function testNewCompiler()
         $this->assertInstanceOf(QueryCompiler::class, $this->driver->newCompiler());
     }
 
+    /**
+     * Test newTableSchema().
+     *
+     * @return void
+     */
+    public function testNewTableSchema()
+    {
+        $actual = $this->driver->newTableSchema('articles');
+        $this->assertInstanceOf(TableSchema::class, $actual);
```

Comment:
Should also include an assertion to check that the table schema has the right table property?